docs/pages/includes/database-access/tctl-auth-sign-3-files.mdx

```diff
@@ -0,0 +1,60 @@
+{{ format="db" }}
+Teleport uses mutual TLS authentication with self-hosted databases. These
+databases must be able to verify certificates presented by the Teleport Database
+Service. Self-hosted databases also need a certificate/key pair that Teleport
+can verify.
+
+By default, the Teleport Database Service trusts certificates issued by a
+certificate authority managed by the Teleport Auth Service. You can either:
+- Configure your self-hosted database to trust this CA, and instruct Teleport to
+  issue a certificate for the database to present to the Teleport Database
+  Service.
+- Configure the Database Service to trust a custom CA.
```

Comment:
I'd prefer tabs here since there are two distinctly separate sets of steps here.
Not a strong preference though - it reads fine if you'd prefer to keep it.

Reply:
I didn't put this in tabs because some pages that include this partial include it within a tab item, and nested tabs don't look great in our docs engine. Otherwise, tabs would make sense.

Reply:
I agree that tabs are much clearer. @kimlisa was reading this preview to fix a WebUI issue on configuring db server with custom CAs and wasn't confused after reading these paragraphs.

Any other way to organize this? 4 tabs instead of the nest tabs? Maybe an extra section on `custom CA` before troubleshooting? Or maybe a separate page on custom CA?

Reply:
BTW, in my opinion, the architecture diagram at the top should also be different for custom CA scenarios. Instead "postgresql certs", it is only database client CA for the PostgreSQL server.

Reply:
Personally, I think `postgresql certs` is vague enough that it could apply to either scenario (e.g., "postgresql client certs"). This is also not ideal for the clarity of the diagram, but I think we can reserve the diagram edits for a separate project, if that makes sense!